scribble/app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,58 @@
+class PostsController < ApplicationController
+  def index
+    @post = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+    render :show
+  end
+
+  def new
+    @post = Post.new
+    render :new
```

Comment:
We do not need these explicit renders, because Rails uses implicit rendering as long as we follow RESTful conventions!